lib/miq_automation_engine/engine/miq_ae_service.rb

```diff
@@ -147,7 +147,7 @@ def instantiate(uri)
     def drb_return(obj)
       # Save a reference to the object, so that we control when it gets deleted.  Otherwise, Ruby Garbage Collection may remove it prematurely.
       # If it is removed prematurely and then referenced by the method, we get a DRb recycled object error
-      @drb_server_references << obj
+      #@drb_server_references << obj
```

Comment:
If this just works, can we remove this method and have instantiate just return `MiqAeServiceObject.new(obj, self)` above?